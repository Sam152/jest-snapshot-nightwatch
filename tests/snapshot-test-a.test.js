const expect = require('expect');
const toMatchSnapshot = require('../index');

expect.extend({toMatchSnapshot});

module.exports = {
  '@unitTest': true,
  'Test using snapshots': () => {
    expect('passing snapshot').toMatchSnapshot();
    expect('second passing snapshot').toMatchSnapshot();
  },
  'Second test case using snapshots': () => {
    expect('third snapshot').toMatchSnapshot();
  }
};
