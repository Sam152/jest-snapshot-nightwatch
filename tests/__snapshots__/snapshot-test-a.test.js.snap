// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Second test case using snapshots 1`] = `"third snapshot"`;

exports[`Test using snapshots 1`] = `"passing snapshot"`;

exports[`Test using snapshots 2`] = `"second passing snapshot"`;
